composer.json

```diff
@@ -29,7 +29,7 @@
         "yiisoft/yii-queue": "*"
     },
     "require-dev": {
-        "phan/phan": "^3.0",
+        "phan/phan": "^5.0",
```

Comment:
We switched phan to psalm. Can remove `phan/phan` dependency and use in CI psalm.